contracts/tests/test_uraiden.py

```diff
@@ -56,6 +57,8 @@ def test_uraiden_init(
         get_uraiden_contract([A, 100])
     with pytest.raises(tester.TransactionFailed):
         get_uraiden_contract([token.address, 0])
+    with pytest.raises(tester.TransactionFailed):
+        get_uraiden_contract([fake_token.address, 100])
```

Comment:
Maybe make the `challenge_period` valid here to emphasize that it's the address that fails and not the challenge period. And make sure to do that in the other calls too, where it's not the challenge period that's supposed to fail but the address.

Reply:
It is valid. Will change it after rebasing the other PR

Reply:
I know it's valid now, but since I saw the other PR too, I just wanted to make sure that you're aware of this. Will approve for now then.